src/libs/ajax.js

```diff
@@ -655,6 +658,34 @@ const Buckets = signal => ({
 })
 
 
+const GoogleBilling = signal => ({
+  listAccounts: async () => {
+    if (!hasBillingScope()) {
+      reportError('Terra does not have access to query your Google Cloud Billing information')
```

Comment:
We generally don't do error reporting at this level, but just let the error propagate out. That gives the consuming code more control over the user experience.

Reply:
Good call. I'll clean that up.

Reply:
@panentheos Do you think we should call `ensureBillingScope` right here?

Reply:
Perhaps. But without a concrete use case it's hard to speculate. These are good building blocks, and as we go forward it will probably become clearer how best to assemble them.